docs/usage/cli.md

```diff
@@ -68,6 +71,81 @@
         └──────────────────────────────────────────────────────────────────────┘
         ```
 
+- Dump the 
+
+    === "Shell"
+
+        ```
+        fr24 flight-list --reg B-HPB
+        fr24 flight-list --flight CX488
+        fr24 flight-list --help
+        ```
+
+    === "Output"
+    
+        ```command
+        $ fr24 flight-list --reg B-HPB
+        Success: Saved 10 rows (1290 bytes) to /home/user/.cache/fr24/flight_list/reg/B-HPB.parquet.
```

Comment:
Honestly... it looks weird to me from a user experience perspective that a tool just caches a result without returning anything. How about printing the flight list to the terminal (as a csv for instance?) in a way it can be piped or something?

Reply:
Yeah, I'll probably make it:
- output the pd.DataFrame repr by default
- `-o`, `--output` to save to a specific directory/filename,
- `-c`, `--cache` to save to the default cache directory

Reply:
I would keep the cache on by default.
`-o` would remove the stdout output? 
I wouldn't go for some pretty repr on stdout as it is hard to pipe to more tools?

Reply:
Sure, maybe:
- `fr24 feed` saves to cache and shows a "preview" of the saved data via `pd.Dataframe` repr
- `fr24 feed --format parquet|csv|json -o /path/to/file` writes to file, no dataframe preview is shown, just how many bytes/records are written
  - `/path/to/file` can be `-` (stdout) and piped to tools, number of bytes/records would then be written to stderr (similar to curl)

Reply:
I would also like to clarify my motivation behind the CLI and more broadly the "core". Usually, when I have to analyse a particular flight, it would involve:
- `fr24.core.flight_list.fetch()` to grab the fr24 flight id, then
- `fr24.core.playback.fetch(flight_id)` to grab the trajectory
- parquet are saved to hardcoded folders
- downstream analyses can then use `.load()` to avoid expensive calls

My intention with the CLI was more of a "download manager" that works with the parquet files. Maybe the "cache" should be more appropriately named "database" 😅

Reply:
Implemented in https://github.com/cathaypacific8747/fr24/pull/24/commits/e19e5ba1d0715f9670aa703227c71b6c1bfc0c4f, CLI docs: https://bf3fe814.flightradar24.pages.dev/usage/cli